src/main/java/org/sonar/plugins/github/MarkDownUtils.java

```diff
@@ -35,7 +35,7 @@
 @InstantiationStrategy(InstantiationStrategy.PER_BATCH)
 public class MarkDownUtils {
 
-  private static final String IMAGES_ROOT_URL = "https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/";
+  private static final String IMAGES_ROOT_URL = "http://sonarsource.github.io/sonar-github/";
```

Comment:
Even though GH uses a proxy, I would have most prolly still set this to `https://`?